### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by ShivanKaul**
Better late than never 😄

### CODE REVIEW

**Comment by grantila** [comment]
Great! But it would be nice to know why major, i.e. what are the breaking changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## 4.0.0
+
```
Comment (awaterma) [inline CHANGELOG.md]:
```suggestion
### Breaking Changes (Major Version)
- Modernized JS Syntax
  - Use ESLint and Prettier to apply consistent, modern formatting
- Support for Promises
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- SameSite cookie support
+- Cookie prefix support 
+- Support for promises
+- Use ESLint and Prettier to apply consistent, modern formatting
+- '.local' support 
+- Numerous bug fixes!
```
Comment (awaterma) [inline CHANGELOG.md]:
```suggestion
### Minor Changes
- SameSite cookie support
- Cookie prefix support 
- '.local' support 
- Numerous bug fixes!
```

**Comment by awaterma** [comment]
@ShivanKaul -- let me know what you think; perhaps there's an even better way to setup our changelog. 

I wonder if we should also consider adding something like this as a Githook?

**Pull request merged**